packages/mdc-feature-targeting/_functions.scss

```diff
@@ -0,0 +1,170 @@
+@import "./variables";
+
+// ==Terminology==
+// Feature:
+//   A simple string (e.g. `color`) representing a cross-cutting feature in
+//   Material.
+// Feature query:
+//   A structure that represents a query for a feature or combination of features. This may be
+//   either a feature or a map containing `op` and `queries` fields. A single feature represents a
+//   simple query for just that feature. A map represents a complex query made up of an operator,
+//   `op`, applied to a list of sub-queries, `queries`.
+//   (e.g. `color`, `(op: any, queries: (color, typography))`).
+// Feature target:
+//   A map that contains the feature being targeted as well as the current feature query. This is
+//   the structure that is intended to be passed to the `@mdc-feature-targets` mixin.
+//   (e.g. `(target: color, query: (op: any, queries: (color, typography))`).
+
+//
+// Public
+//
+
+// Creates a feature target from the given feature query and targeted feature.
+@function mdc-feature-create-target($feature-query, $targeted-feature) {
+  $feature-target: (target: $targeted-feature, query: $feature-query);
```

Comment:
Reverse this in order of function's arguments.

`(query: $feature-query, target: $targeted-feature)`